installer/upgrader/runTranslationTest.php

```diff
@@ -33,3 +33,4 @@
 $translate->up('pim');
 $translate->up('leave');
 $translate->up('time');
+$translate->up('maintenance');
```

Comment:
Will have to do the same in runTranslations.php as well

Reply:
A good job on the translations !

Reply:
added